docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,71 @@
+Canonical URLs
+==============
+
+A `Canonical URL`_
+allows you to specify the preferred version of a web page to prevent duplicated content.
+They are mainly used by search engines to link users to the correct
+version and domain of your documentation.
+
+If canonical URL's aren't used,
+it's easy for outdated documentation to be the top search result for various pages in your documentation.
+This is not a perfect solution for this problem,
+but generally people finding outdated documentation is a big problem,
+and this is one of the suggested ways to solve it from search engines.
```

Comment:
This feels redundant, and/or isn't adding much for the reader. Is there a specific reason worth mentioning why it isn't a great solution perhaps? If not, maybe worth skipping this sentence

```suggestion
```

Reply:
Agreed, but the sentence before describes the problem better. This sentences mostly distills down to "this is one imperfect solution to the problem we just outlined", and so sounds redundant, and isn't confident.

I'm suggesting either expanding on the problem/solution or dropping this sentence. Something like this expands on both:

```suggestion
Linking to outdated documentation is confusing for your readers,
but this can be avoided by telling search engines exactly where to find your documentation.
```